application/browser/application_system_tizen.cc

```diff
@@ -90,24 +108,14 @@ bool ApplicationSystemTizen::LaunchFromCommandLine(
   // Handles raw app_id passed as first non-switch argument.
   const base::CommandLine::StringVector& args = cmd_line.argv();
   CHECK(!args.empty());
+  launch_params_ = const_cast<base::CommandLine*>(&cmd_line);
   base::FilePath exec_path(args[0]);
 
   std::string app_id = exec_path.BaseName().MaybeAsASCII();
-  LOG(INFO) << "Attempting to launch the app with id: " << app_id;
-  if (!IsValidApplicationID(app_id))
-    return false;
-
-  ApplicationServiceTizen* app_service_tizen =
-      ToApplicationServiceTizen(application_service_.get());
-
-  Application* app = app_service_tizen->LaunchFromAppID(app_id);
-
-  if (!app)
-    return false;
 
   const std::string& name = std::string("xwalk-") + app_id;
   appcore_ops.cb_app = application_event_cb;
-  appcore_ops.data = app;
+  appcore_ops.data = this;
```

Comment:
Ok, I can do this that way. I thought about it but idea of passing ownership of some ptr to handler didn't appeal to me. On the second though, this can be deallocated on terminate event.